### PR TITLE
feat: adding exec ed to csv

### PR DESCRIPTION
**Pull request by kiram15**
## Description

Adding Executive Education information to the csv download on explore catalog. 

## Ticket Link

[Associated Ticket
](https://2u-internal.atlassian.net/browse/ENT-6421)

## Post-review

* [ ] Squash commits into discrete sets of changes
* [ ] Ensure that once the changes have been [deployed to stage](https://gocd.tools.edx.org/go/pipeline/activity/stage-enterprise_catalog), prod is manually deployed

### CODE REVIEW

File: enterprise_catalog/apps/api/v1/export_utils.py [hunk 1]
```diff
@@ -61,28 +61,51 @@
...
+    'What You’ll Learn',
+    'Full Description',
+]
+
 ALGOLIA_ATTRIBUTES_TO_RETRIEVE = [
-    'title',
-    'key',
-    'content_type',
-    'partners',
     'advertised_course_run',
```
Comment (kiram15) [inline enterprise_catalog/apps/api/v1/export_utils.py:85]: It was hard to look through all of these so I just alphabetized them!

**Review by johnnagro — APPROVED** [review]

File: enterprise_catalog/apps/api/v1/export_utils.py [hunk 2]
```diff
@@ -119,6 +142,7 @@ def course_hit_to_row(hit):
     """
     Helper function to construct a CSV row according to a single Algolia result course hit.
     """
+    exec_course = hit.get('course_type') == 'executive-education-2u' if True else False
```
Comment (marlonkeating) [inline enterprise_catalog/apps/api/v1/export_utils.py]: Would make the variable name more clear that it's a boolean, and also remove the extraneous logic at the end.
```suggestion
    is_exec_course = hit.get('course_type') == 'executive-education-2u'
```

**Review by marlonkeating — CHANGES_REQUESTED** [review]
question: For course_hit_to_row(), given the number of exceptions for executive courses, might the logic be easier to follow if we structured the function like
`if is_exec_course:
    # Parse everything for executive course
  else:
    # Parse everything for non-executive course
  
  `

File: enterprise_catalog/apps/api/v1/export_utils.py (hunk 2 above)
Comment (kiram15) [reply enterprise_catalog/apps/api/v1/export_utils.py]: Yeah this is what we were talking about, basically its the question of do we want a lot of copied code to be cleaner and more readable, or do we want to reduce copied code. I agreed with you, but John and Alex were in the nay boat. Tbh its just preference 
https://twou.slack.com/archives/C049WNMFFRP/p1669755717967509

**Review by marlonkeating — APPROVED** [review]

File: enterprise_catalog/apps/api/v1/export_utils.py (hunk 2 above)
Comment (alex-sheehan-edx) [reply enterprise_catalog/apps/api/v1/export_utils.py]: I have no objections to change the variable name!

**Review by alex-sheehan-edx — APPROVED** [review]
lgtm!

**Pull request merged**